packages/components/src/components/modal/modal.jsx

```diff
@@ -17,13 +17,13 @@ class ModalElement extends React.PureComponent {
     }
 
     componentDidMount = () => {
-        document.addEventListener('mousedown', this.handleClickOutside);
+        if (this.props.is_closed_on_outside_click) document.addEventListener('mousedown', this.handleClickOutside);
```

Comment:
```suggestion
        if (this.props.is_closed_on_outside_click) { 
             document.addEventListener('mousedown', this.handleClickOutside);
        }
```

Reply:
Because it's a different flow for the `Set Currency Modal` , we want to encourage client to `Set Currency` before proceeding instead of closing the popup dialog / modal.